DOCS/man/options.rst

```diff
@@ -1248,9 +1248,9 @@ Video
     special values:
 
     :no:        always use software decoding (default)
+    :auto-safe: enable any whitelisted hw decoder (see below)
     :auto:      forcibly enable any hw decoder found (see below)
     :yes:       exactly the same as ``auto``
```

Comment:
should "yes" be changed to mean auto-safe too?

Reply:
It's something of a configuration compatibility break, although I think it's a reasonable thing to do, so that's why I didn't change it, but happy to do so.

Reply:
Added another commit for this.